src/app/collections/Mod.tsx

```diff
@@ -21,7 +31,7 @@ interface Props {
   onAnItem: boolean;
 }
 
-export default function Collectible({ inventoryItem, defs, buckets, owned, onAnItem }: Props) {
+export default function ModCollectible({ inventoryItem, defs, buckets, owned, onAnItem }: Props) {
```

Comment:
Wow, this was my mistake originally. This default export should match the filename, so `Mod`.